robot.py

```diff
@@ -273,7 +275,7 @@ def disabledPeriodic(self) -> None:
             self.status_lights.vision_timeout()
         self.status_lights.execute()
 
-        if self.gamepad.getAButtonPressed():
+        if not self.coast_button:

```

Comment:
This is using a `DigitalInput` object in a boolean context. A DigitalInput object will always be truthy, so this condition will always be false.

A `.get()` is missing here, but you _also_ need some sort of debounce otherwise this'll constantly toggle, which is certainly not what you want.